src/Uno.UI/Uno.UI.csproj

```diff
@@ -153,6 +153,8 @@
 		<Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
 	</ItemGroup>
 	<ItemGroup>
+		<Folder Include="Microsoft\UI\Xaml\Controls\Common\" />
```

Comment:
Remove the changes to Uno.UI.csproj from the PR (and you can safely revert them locally too) - they get added when you include files via Visual Studio, but really they're unnecessary, because files are picked up by globbing.

Reply:
Yes, I noticed that when I imported the Strings folder it just popped up automatically in the solution. I'll be sure to clean this up!

Reply:
Concerning `InvalidOperationException`, I debated using that one or 
`InvalidEnumArgumentException`. Both don't fit 100% so I went with the generic base Exception with a note to take another look. I'll be happy to switch it, thanks for the recommendation.